src/relentless/simulate/hoomd.py

```diff
@@ -1228,11 +1240,27 @@ def act(self, timestep):
                         for i, j in self._rdf:
                             query_args = dict(self._rdf[i, j].default_query_args)
                             query_args.update(exclude_ii=(i == j))
+                            neighbors = (
+                                aabbs[j]
+                                .query(
+                                    snap.particles.position[type_masks[i]], query_args
```

Comment:
@mphoward , I think this should be type `j` since aabbs above is calculated for `i`